src/main.cpp

```diff
@@ -183,6 +183,35 @@ namespace rhost {
 #endif
     }
 
+    void add_dir_to_loader_path(fs::path &r_dir) {
+#ifdef _WIN32
+        wchar_t buff[32767] = {}; // max buffer size for environment variable values on windows.
+        int nchars = GetEnvironmentVariable(L"PATH", buff, ARRAYSIZE(buff));
+        HRESULT err = S_OK;
+        if (nchars == 0) {
+            err = HRESULT_FROM_WIN32(GetLastError());
+            logf(log_verbosity::minimal, "Failed to get PATH environment variable [GetEnvironmentVariable]: %x\n", err);
```

Comment:
It is, of course, unusual, but technically PATH variable doesn't have to exist...